### PR TITLE
feat: ui components for create-wallet

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: packages/ui/src/assets/images/common/green-tick.svg
Comment (irshadCypherock) [inline packages/ui/src/assets/images/common/green-tick.svg:1]: I think this already exists in the repo

File: packages/ui/src/components/atoms/Container.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export const Container: FC<ContainerProps> = ({ children, ...props }) => (
 );
 
 Container.defaultProps = {
-  children: null,
+  children: undefined,
```
Comment (irshadCypherock) [inline packages/ui/src/components/atoms/Container.tsx]: children should be `null`

File: packages/ui/src/components/atoms/Typography.tsx [hunk 2]
```diff
@@ -192,7 +194,7 @@ export const Typography = ({
 
 Typography.defaultProps = {
   variant: 'p',
-  children: null,
+  children: undefined,
```
Comment (irshadCypherock) [inline packages/ui/src/components/atoms/Typography.tsx]: children in react can be JSX for null

File: packages/ui/src/components/molecules/AlertBox.tsx [hunk 3]
```diff
@@ -11,7 +17,7 @@ export interface AlertBoxProps
   extends UtilsProps,
     React.ButtonHTMLAttributes<HTMLDivElement> {
   alert: string;
-  imageSrc?: string;
+  icon?: React.JSX.Element;
```
Comment (irshadCypherock) [inline packages/ui/src/components/molecules/AlertBox.tsx]: Can we use `React.ReactNode` ?

File: packages/ui/src/components/molecules/Layouts/OnboardingLayout.tsx
Comment (irshadCypherock) [inline packages/ui/src/components/molecules/Layouts/OnboardingLayout.tsx:1]: Remove the duplicate `OnboardingLayout.tsx` file outside `Layouts` folder

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

File: packages/ui/src/assets/images/common/green-tick.svg
Comment (ghost) [reply packages/ui/src/assets/images/common/green-tick.svg:1]: slightly different size and width, we could merge them in future

**Review by irshadCypherock — APPROVED** [review]

**Pull request merged**